src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -1068,16 +1068,14 @@ private Matches() {}
   }
 
   /**
-   * Does NOT check for Canals, Blitzing, Loading units on transports, TerritoryEffects that disallow units, Stacking
-   * Limits, Unit movement
-   * left, Fuel available, etc.<br>
+   * Does NOT check for: Canals, Blitzing, Loading units on transports, TerritoryEffects that disallow units, Stacking
+   * Limits, Unit movement left, Fuel available, etc.
```

Comment:
If the changes I made in #2297 look like they're going to conflict, feel free to merge this PR first, as I think it will be easier for me to resolve the conflicts over there.

Reply:
I'll take a look to see if I think they'll conflict. Hopefully not but either way it shouldn't be too bad to resolve since my changed are pretty localized.